Terraform/AWS/task3/variables.tf

```diff
@@ -0,0 +1,3 @@
+variable env_code{
+    type = string
+}
```

Comment:
Add newline at the end of the file.

Reply:
Hi @geoffrey1330 can you explain why should I add a new line at the end of each file

Reply:
It is a requirement by git for a good code style.

Reply:
okay thank you